cloud-common/src/main/java/com/sequenceiq/cloudbreak/cloud/store/InMemoryStateStore.java

```diff
@@ -1,27 +1,40 @@
 package com.sequenceiq.cloudbreak.cloud.store;
 
+import com.sequenceiq.cloudbreak.cloud.scheduler.PollGroup;
+
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
-import com.sequenceiq.cloudbreak.cloud.scheduler.PollGroup;
-
 public final class InMemoryStateStore {
 
-    private static final Map<Long, PollGroup> STATE_STORE = new ConcurrentHashMap<>();
+    private static final Map<Long, PollGroup> STACK_STATE_STORE = new ConcurrentHashMap<>();
+    private static final Map<Long, PollGroup> CLUSTER_STATE_STORE = new ConcurrentHashMap<>();
```

Comment:
I think this is the fastest way to make Mesos/Marathon related cluster pollers cancelable. I agree this is not the best way as we discussed it last week, but to make the refactor that use a common and well-designed polling mechanism takes much more time to develop.
Without this fix mesos/marathon cluster installation could not be cancelled, when error occurs at container creation the failed container has not be removed from orchestration side.